src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -250,14 +250,16 @@ private static ProcessModuleCollection GetModules(int processId, bool firstModul
 
     internal static partial class NtProcessInfoHelper
     {
+        private const int AllProcessIds = -1;
```

Comment:
are valid PID's always signed? On Windows -1 is [safe](https://blogs.msdn.microsoft.com/oldnewthing/20040223-00/?p=40503), on Linux kill takes `-1` to mean all processes so I assume they are. On Mac probably the same but mine isn't powered up right now to chck.

Reply:
.. and just remembered this is only for Windows. So per Raymond's blog, you're good

Reply:
UAP and UAPAOT Configurations don't build this file, which is why CI is failing. You can try moving this const to the other file.